sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/coverage/CxxCoverageSensor.java

```diff
@@ -165,7 +165,9 @@ private void saveMeasures(SensorContext context,
       String filePath = entry.getKey();
       InputFile cxxFile = fs.inputFile(fs.predicates().hasPath(filePath));
       if (cxxFile != null) {
-        Collection<Measure> measures = entry.getValue().createMeasures();
+        CxxUtils.LOG.debug("Saving for file '{}'", filePath);
+        CoverageMeasuresBuilder builder = entry.getValue();
```

Comment:
Sure but can't we print it in the exception handler. Now for nominal cases logs are going to be really big. We have done some changes to the plugin to keeps things to a minimal and this seems going the other team around